### ISSUE TITLE
Operation aborted (Exception from HRESULT: 0x80004004 (E_ABORT))

### ISSUE THREAD

**Issue by oddrationale** [phw198/OutlookGoogleCalendarSync#227]
Hello!

When I unzip the portable version and run the executable, the splash window shows up and a web browser is opened to the site. But the application does not start up. Looking through the OGcalsync.log file, I came across the following error message:

> 2017-06-01 15:20:51,583 ERROR 1 OutlookGoogleCalendarSync.OGCSexception [0] - System.Runtime.InteropServices.COMException: Operation aborted (Exception from HRESULT: 0x80004004 (E_ABORT))
2017-06-01 15:20:51,583 ERROR 1 OutlookGoogleCalendarSync.OGCSexception [0] - Code: 0x80004004;-2147467260

I've attached the full log here: [OGcalsync.log.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/1045949/OGcalsync.log.txt)

Here are my system information:
OS: Windows 7 Enterprise SP1
OGCS Version: v2.4.0 Beta (Portable zip version)
Outlook Version: MS Outlook 2013 (Office Professional Plus 2013)

I have a suspicion that this might have something to do with Outlook security settings in the Trust Center. Maybe the Programmatic Access settings? Unfortunately, these settings are locked down by my organization and I cannot change it to experiment.

Let me know how else I can help troubleshoot the issue. Thanks!

Best,
Dariel Dato-on

**Comment by phw198** [comment]
Unfortunately I think you're right - log file shows:
```
2017-06-01 15:20:51,547 INFO 1 OutlookGoogleCalendarSync.OutlookNew [0] - Exchange connection mode: olCachedConnectedFull
2017-06-01 15:20:51,550 WARN 1 OutlookGoogleCalendarSync.OutlookNew [0] - We seem to have a faux connection to Outlook! Forcing starting it with a system call :-/
```
What should happen after it has connected to Exchange is work out what your account's email address is, but it's not being allowed to do this.

If you can't change your Outlook security settings I'm not sure there's much you can do :disappointed: 

I'm guessing not which is why you're using the Portable ZIP version, but do you have rights to start applications as Administrator? If so, try starting both Outlook and OGCS as Administrator...

**Comment by phw198** [comment]
Duplicate of #268 which has been resolved in v2.6.1
